iChef-WebApi/src/main/java/com/kitchen/iChef/Repository/Interfaces/ICrudRepository.java

```diff
@@ -0,0 +1,18 @@
+package com.kitchen.iChef.Repository.Interfaces;
+
+import java.util.List;
+
+
+public interface ICrudRepository<E, ID> {
```

Comment:
I am not sure what the java standard is for documenting code but in C# you have documentation for an operation on the interface and for the implementation it can be left empty. Maybe we should do that here too? We need to know what exceptions can be thrown if any and what will be returned if no user is found for example

Reply:
the Exception handling is set on another ticket and the documentation will be made after that